regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -96,7 +96,6 @@ var CommentView = Backbone.View.extend({
     this.model = blank ?
       new CommentModel(options) :
       comments.get(section) || new CommentModel(options);
-    this.listenTo(this.model, 'destroy', this.setSection.bind(this, section, tocId, label, true));
```

Comment:
Derp. I totally tried arguments but didn't seem to work for me at the time. Maybe just needed to step away and recompile again and now fixed without having to remove. TY!